src/index.tsx

```diff
@@ -1,9 +1,12 @@
-import React from 'react';
+// import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter as Router } from 'react-router-dom';
```

Comment:
```suggestion
import { HashRouter as Router } from 'react-router-dom';
```
Використовцй hashrouter, щоб все працювало коректно.
Зараз по демо лінку отримаємо 404 помилку, якщо, наприклад, оновимо сторінку або введемо URL вручну